LkeServices/Blockchain/BlockchainExplorersProvider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Core.Blockchain;
+using Lykke.HttpClientGenerator.Caching;
+using Lykke.Service.BlockchainSettings.Client;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LkeServices.Blockchain
+{
+    public class BlockchainExplorersProvider : IBlockchainExplorersProvider
+    {
+        private readonly IBlockchainSettingsClient _client;
+        private readonly IClientCacheManager _clientCacheManager; //Required to Invalidate Cache on request
+
+        public BlockchainExplorersProvider(Lykke.Service.BlockchainSettings.Client.IBlockchainSettingsClient client,
+            IClientCacheManager clientCacheManager)
+        {
+            _client = client;
+            _clientCacheManager = clientCacheManager;
+        }
+
+        public async Task<IEnumerable<BlockchainExplorerLink>> GetAsync(string blockchainType, string txHash)
+        {
+            var explorers = await _client.GetBlockchainExplorerByTypeAsync(blockchainType);
```

Comment:
Is this call cached?

Reply:
![image](https://user-images.githubusercontent.com/14955144/53013082-49214c80-345e-11e9-92e8-7eb4cca31398.png)